tokens.json

```diff
@@ -1819,6 +1855,40 @@
         "value": "#4A4F62A6",
         "description": "Overlay light for modals and drawer",
         "type": "color"
+      },
+      "gradients": {
+        "primary": {
+          "value": "linear-gradient(0deg, #151A2D 0%, #4F0599 100%);",
+          "type": "color"
+        },
+        "blue": {
+          "value": "linear-gradient(200.14deg, #0092FF 13.73%, #4F0599 86.59%);",
+          "type": "color"
+        },
+        "purple": {
+          "value": "linear-gradient(197.81deg, #8430D9 12.89%, #4F0599 87.84%);",
+          "type": "color"
+        },
+        "pink": {
+          "value": "linear-gradient(200.94deg, #FF0CD2 14.71%, #4F0599 86.16%);",
+          "type": "color"
+        },
+        "green": {
+          "value": "linear-gradient(202.33deg, #45D29F 15.16%, #4F0599 85.44%);",
+          "type": "color"
+        },
+        "aqua": {
+          "value": "linear-gradient(199.45deg, #03D5E0 14.03%, #4F0599 88.52%);",
+          "type": "color"
+        },
+        "fuschia": {
+          "value": "linear-gradient(200.02deg, #FF1C6E 13.34%, #4F0599 86.64%);",
+          "type": "color"
+        },
+        "orange": {
+          "value": "linear-gradient(201.62deg, #FF5D2A 14.62%, #4F0599 85.81%);",
```

Comment:
Possible not to have `;` at the end of them?

Reply:
just pushed a new request, can you check ?